src/components/Book.js

```diff
@@ -7,7 +7,7 @@ import { deleteBook } from '../redux/books/books';
 const Book = ({ books }) => {
   const dispatch = useDispatch();
   return (
-    <ul>
+    <ul className="book-list">
       {books.map((book) => (
         <li key={uuidv4()}>
           <span>{book.title}</span>
```

Comment:
- [ ] Following the project requirements, Styling should be  EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/).

- Your design:
<img width="959" alt="image" src="https://user-images.githubusercontent.com/77190203/173200287-32e5be6e-e3b2-4f77-ba07-1b45cb52ee4c.png">

- Zeplin design:
![](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/images/bookstore.png?raw=true)